ccd-definition/CaseEventToFields/addStatementOfService.json

```diff
@@ -0,0 +1,47 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "addStatementOfService",
+    "CaseFieldID": "c9Declaration",
```

Comment:
Should this not be statementOfServiceDeclaration to be consistent with the serviceDeclarationLabel?

Reply:
I think the event is to add the statement of service and not just the declaration and so I think it should remain addStatementOfService as addStatementOfServiceDeclaration makes me think we are only submitting the declaration and not the event.